src/pages/elements/typography/type-sets.mdx

```diff
@@ -111,7 +111,7 @@ headings are needed.
 
 The fluid heading styles are primarily used in web pages, and are therefore part
 of the expressive set of type styles. These headings are responsive and the type
-styles change size at different breakpoints.
+styles change based on the viewport width.
```

Comment:
```suggestion
styles change size between the different breakpoints, based on the viewport width.
```

This is a hard behavior to describe, this might be a little more clear about how the fluid type tokens behave.